Sources/Datadog/DatadogInternal/Context/DatadogContext.swift

```diff
@@ -78,4 +78,10 @@ internal struct DatadogContext {
     /// This value can be `nil` of no service is currently registered, or if the device does
     /// not support telephony services.
     var carrierInfo: CarrierInfo?
+
+    /// `true` if the Low Power Mode is enabled.
+    var batteryStatus: BatteryStatus?
```

Comment:
The comment is wrong. It's worth explaining in the comment when to expect `nil` value, like we did for `networkConnectionInfo` and `carrierInfo` 👌.

Reply:
Good catch! I've improved this doc and added some for the introduced reader and publisher.